app/models/va_health_facility.rb

```diff
@@ -14,7 +14,7 @@ def self.query(bbox:, services:)
 
   def self.find_by_id(id:)
     results = client.get(identifier: id)
-    VAHealthFacility.from_gis(results.first) unless results.nil?
+    VAHealthFacility.from_gis(results.first) unless results.nil? || results.empty?
```

Comment:
blank? is a Rails method that will combine these two 
http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F